common/events.go

```diff
@@ -115,7 +122,7 @@ const (
 	SEC_MOD_COMPLETE_EVENT
 	SEC_MOD_REJECT_EVENT // 95
 
-	FIVE_GMM_STATUS_EVENT = UE_5GS_MOBILITY_MANAGEMENT_EVENTS + 14 + iota // 86
+	FIVE_GMM_STATUS_EVENT = 100 + iota // 100 
```

Comment:
The following minor change will fix the linter issue
```suggestion
	FIVE_GMM_STATUS_EVENT = 100 + iota // 100
```